sycl/include/sycl/ext/intel/esimd/detail/types.hpp

```diff
@@ -357,14 +356,14 @@ std::enable_if_t<is_clang_vector_type_v<To> && is_clang_vector_type_v<From>, To>
 template <typename U> constexpr bool is_type() { return false; }
```

Comment:
BTW, we have type checker with exactly same definition, but a better name:
https://github.com/intel/llvm/blob/sycl/sycl/include/sycl/ext/intel/experimental/esimd/detail/util.hpp#L32